_episodes/13-dplyr.md

```diff
@@ -388,16 +388,18 @@ starts.with <- substr(gapminder$country, start = 1, stop = 1)
 # Filter countries that start with "A" or "Z"
 az.countries <- gapminder[starts.with %in% c("A", "Z"), ]
 # Make the plot
-ggplot(data = az.countries, aes(x = year, y = lifeExp, color=continent)) +
+ggplot(data = az.countries, aes(x = year, y = lifeExp, color = continent)) +
   geom_line() + facet_wrap( ~ country)
 ~~~
 {: .r}
 
-This code makes the right plot but it also creates some variable (`starts.with`
+This code makes the right plot but it also creates some variables (`starts.with`
 and `az.countries`) that we might not have any other uses for. Just as we used
-`%>%` to pipe data into `dplyr` we can use it to pass data to `ggplot`. By
-combining `dplyr` and `ggplot` we can make the same figure without creating any
-new variables or modifying the data.
+`%>%` to pipe data along a chain of `dplyr` functions we can use it to pass data
+to `ggplot`. Because `%>%` replaces the first argument in a function we don't
```

Comment:
I would change this from "to `ggplot`" to "to `ggplot()`". It may seem overly pedantic, but we try to stick to a consistent code style whenever possible. The idea is that when we are referring to the function we append "()" and when we are referring to the package we leave off the parentheses. Thus no parentheses in "`dplyr` functions".  Parentheses should be added after `ggplot` on line 400 as well, but not on line 401 (following the above logic).